web/src/engine/websites/ComicK.ts

```diff
@@ -146,9 +141,7 @@ export default class extends DecoratableMangaScraper {
     }
 
     public override async FetchPages(chapter: Chapter): Promise<Page[]> {
-        const uri = new URL('/chapter/' + chapter.Identifier, this.apiUrl);
-        const request = new Request(uri.href);
-        const data = await FetchJSON<APIChapter>(request);
-        return data.chapter.md_images.map(image => new Page(this, chapter, new URL(`https://meo.comick.pictures/${image.b2key}`), { Referer: this.URI.href }));
+        const { chapter: { images } } = await FetchJSON<APIPages>(new Request(new URL(`/chapter/${chapter.Identifier}?tachiyomi=true`, this.apiUrl)));
```

Comment:
Remove the `tachiyomi` parameter.
 This is an identifier that basically tells the website owner:

 > Hi, me (IP address X.X.X.X) is using a software to bypass your ads

The website owner may apply measures against such IP addresses...

Reply:
It will be a nightmare to find the good CDN without that and it will result errors 404 errors. Its the main reason why this pr was done.

Reply:
Done